live-build/variants/internal-dcenter/package-lists/dcenter.list.chroot

```diff
@@ -24,8 +24,10 @@ libldap2-dev
 libsasl2-dev
 nfs-kernel-server
 openjdk-11-jdk-headless
-python3
-python3-dev
+python3.12
+python3.12-dbg
+python3.12-dev
 python3-venv
+python3.12-venv
```

Comment:
This line looks duplicated

Reply:
@jtamagnan-delphix ,
You asked me earlier to keep both python3-venv and python3.12-venv?

Reply:
Apologies I thought I had read `python3.12-venv` twice